edgedb/protocol/protocol.pyx

```diff
@@ -132,7 +132,7 @@ cdef class SansIOProtocol:
 
         self.server_settings = {}
         self.reset_status()
-        self.protocol_version = (PROTO_VER_MAJOR, 0)
+        self.protocol_version = (PROTO_VER_MAJOR, PROTO_VER_MINOR)
```

Comment:
I think this is bad to assume highest protocol version before negotiation is done. And I don't see any need to change it in this PR.

Reply:
changed